core-java-modules/core-java-11-2/src/test/java/com/baeldung/features/HttpClientIntegrationTest.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.features;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockserver.integration.ClientAndServer;
+import org.mockserver.model.HttpStatusCode;
+import org.mockserver.socket.PortFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockserver.integration.ClientAndServer.startClientAndServer;
+
+class HttpClientIntegrationTest {
+
+    private static ClientAndServer mockServer;
+    private static int port;
+
+    @BeforeAll
+    static void startServer() {
+        port = PortFactory.findFreePort();
+        mockServer = startClientAndServer(port);
+        mockServer.when(new org.mockserver.model.HttpRequest().withMethod("GET"))
```

Comment:
two space indents when continuing a line

Reply:
is there a code formatter for intelij?

Reply:
https://github.com/eugenp/tutorials/blob/master/intelliJ/intelliJ-formatter.xml